modules/sbo-api-differences.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/understanding-service-binding-operator.adoc
+
+:_content-type: CONCEPT
+[id="sbo-api-differences_{context}"]
+= API differences
+
+The CRD of the {servicebinding-title} supports the following APIs:
+
+* *Service Binding* with the `binding.operators.coreos.com` API group.
+* *Service Binding (Spec API)* with the `servicebinding.io` API group.
```

Comment:
I'm not sure about these bold references to service binding. I believe I commented on this issue in another PR I just reviewed for you. Please apply the same changes that you applied to that PR here in this PR and in any other similar instances you're working on. Thanks!